run.py

```diff
@@ -13,6 +13,6 @@
 logger = wrap_logger(logging.getLogger(__name__))
 
 if __name__ == '__main__':
-    port = os.getenv('FS_DEV_PORT', 8080)
+    port = os.getenv('FS_DEV_PORT', 8082)
```

Comment:
This is doing something different to the other repos and not using the config for the port number. Is that ok?

Reply:
This is something left over from way back when frontstage was started, would be good to bring it in line with the other repos

Reply:
Attached a simple patch for [using config](https://github.com/ONSdigital/ras-frontstage/files/1606695/use_config.txt).